src/server/game/DataStores/DB2Stores.cpp

```diff
@@ -1621,6 +1626,10 @@ uint32 DB2Manager::LoadStores(std::string const& dataPath, LocaleConstant defaul
             sOldContinentsNodesMask[field] |= submask;
     }
 
+    for (PVPScoreboardLayoutEntry const* layout : sPVPScoreboardLayoutStore)
```

Comment:
Why bother loading the Layout db2 when you are only using data from PvPStat?

Reply:
The idea was to only load the ones that are used in a layout. But this is kinda misleading with the name of the property and they have the same amount of records anyway. Will remove it